controllers/cache/redis.go

```diff
@@ -65,17 +70,18 @@ func (redis *RedisReconciler) Delete() (*lcm.CRStatus, error) {
 }
 
 func (redis *RedisReconciler) Scale() (*lcm.CRStatus, error) {
-	panic("implement me")
-}
-
-func (redis *RedisReconciler) ScaleUp(newReplicas uint64) (*lcm.CRStatus, error) {
-	panic("implement me")
-}
-
-func (redis *RedisReconciler) ScaleDown(newReplicas uint64) (*lcm.CRStatus, error) {
-	panic("implement me")
+	if err := redis.ScaleUpCache(); err != nil {
```

Comment:
Don't we need to check function to determine if it's up or down?

Reply:
We need to do different operations on different actions. If scale down, we need to check whether the node to be destroyed is the master node.